server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -372,4 +377,35 @@ public void validateHealthChecksShouldReturnOk() {
         .then()
             .statusCode(HttpStatus.OK_200);
     }
+
+    @Test
+    public void cassandraMappingsEndpointShouldCleanAndRepopulateMappingsSourcesProjectionTable() {
```

Comment:
it's not what is tested. Here, you just test that calling solveInconsistencies results in the same state as before, right ?